Jenkinsfile

```diff
@@ -121,12 +121,15 @@ def rpm_scan_test = '''lmd_src=\\\"maldet-current\\\"
                        popd
                        sudo freshclam
                        rm -f /var/tmp/clamscan.out
+                       rm /var/tmp/\\\${lmd_tarball}
+                       rm -rf /var/tmp/\\\${lmd_src}
                        sudo clamscan -d /usr/local/maldetect/sigs/rfxn.ndb \
                                 -d /usr/local/maldetect/sigs/rfxn.hdb -r \
                                 --exclude-dir=/usr/local/maldetect \
                                 --exclude-dir=/usr/share/clamav \
-                                --infected /etc /usr | \
-                         tee /var/tmp/clamscan.out
+                                --exclude-dir=/var/lib/clamav \
+                                --exclude-dir=/sys \
```

Comment:
What about `/proc` and `/dev`.  I'm surprised not excluding those isn't completely fouling up scanning, TBH.

Reply:
Added skips for /proc and /dev and merged with current master.